front/app/containers/MainHeader/utils.ts

```diff
@@ -0,0 +1,42 @@
+import { Locale } from '@citizenlab/cl2-component-library';
+import { locales } from 'containers/App/constants';
+import { includes } from 'lodash-es';
+import { isPage } from 'utils/helperUtils';
+
+export const isIdeaPage = (urlSegments: string[]) => {
+  const firstUrlSegment = urlSegments[0];
```

Comment:
Moved all of these checks into a shared utils file as both the MainHeader and ScrollHeader use them to determine whether the header should be shown on specific pages.